diff --git a/node_modules/ai/dist/index.js b/node_modules/ai/dist/index.js
index 51dc17f..9b64f88 100644
--- a/node_modules/ai/dist/index.js
+++ b/node_modules/ai/dist/index.js
@@ -1505,17 +1505,17 @@ async function standardizePrompt({
       );
     }
   }
-  const validationResult = await (0, import_provider_utils6.safeValidateTypes)({
-    value: messages,
-    schema: import_v46.z.array(modelMessageSchema)
-  });
-  if (!validationResult.success) {
-    throw new import_provider19.InvalidPromptError({
-      prompt,
-      message: "The messages must be a ModelMessage[]. If you have passed a UIMessage[], you can use convertToModelMessages to convert them.",
-      cause: validationResult.error
-    });
-  }
+  // const validationResult = await (0, import_provider_utils6.safeValidateTypes)({
+  //   value: messages,
+  //   schema: import_v46.z.array(modelMessageSchema)
+  // });
+  // if (!validationResult.success) {
+  //   throw new import_provider19.InvalidPromptError({
+  //     prompt,
+  //     message: "The messages must be a ModelMessage[]. If you have passed a UIMessage[], you can use convertToModelMessages to convert them.",
+  //     cause: validationResult.error
+  //   });
+  // }
   return { messages, system };
 }
 
