diff --git a/node_modules/ai/dist/index.js b/node_modules/ai/dist/index.js
index ee21dba..d250979 100644
--- a/node_modules/ai/dist/index.js
+++ b/node_modules/ai/dist/index.js
@@ -2584,17 +2584,17 @@ async function standardizePrompt({
       message: "System messages are not allowed in the prompt or messages fields. Use the instructions option instead."
     });
   }
-  const validationResult = await safeValidateTypes({
-    value: messages,
-    schema: z.array(modelMessageSchema)
-  });
-  if (!validationResult.success) {
-    throw new InvalidPromptError2({
-      prompt,
-      message: "The messages do not match the ModelMessage[] schema.",
-      cause: validationResult.error
-    });
-  }
+  // const validationResult = await safeValidateTypes({
+  //   value: messages,
+  //   schema: z.array(modelMessageSchema)
+  // });
+  // if (!validationResult.success) {
+  //   throw new InvalidPromptError2({
+  //     prompt,
+  //     message: "The messages do not match the ModelMessage[] schema.",
+  //     cause: validationResult.error
+  //   });
+  // }
   return { messages, instructions };
 }
 
